### DISCUSSION TITLE
Can you add sorting for Overlap?

### DISCUSSION THREAD

**Discussion by MusicDendy** [zBritva/charticulator-visual-community#16] [Ideas]
Can you add sorting for Overlap?
![image](https://github.com/zBritva/charticulator-visual-community/assets/24212122/85bb89bd-5ddf-49d6-8d2f-d6a6e0cc15d1)
Like we have for other Sub-Layout
![image](https://github.com/zBritva/charticulator-visual-community/assets/24212122/2e8a529f-b486-4851-b529-d941749c4a42)

**Comment by zBritva** [comment]
@MusicDendy Thanks! I will consider the implementation after passing Power BI Certification.
